ui/raidboss/data/05-shb/dungeon/dohn_mheg.js

```diff
@@ -133,6 +147,8 @@
       id: 'Dohn Mheg Crippling Blow',
       regex: / 14:35A4:Aenc Thon, Lord [Oo]f [Tt]he Lengthsome Gait starts using Crippling Blow on (\y{Name})/,
       regexDe: / 14:35A4:Aenc Thon der Langbeinige starts using Verkrüppelnder Schlag on (\y{Name})/,
+      regexFr: / 14:35A4:Aenc Thon le virtuose starts using Coup handicapant on (\y{Name})/,
```

Comment:
I wonder if some of this needs to be capitalized, but maybe somebody who is playing with the French version can confirm.

Reply:
dont get me started, from all Fr ones i could find, random things are always capitalized (compared to the ones i can create from games data)